.travis.mk

```diff
@@ -295,7 +295,7 @@ test_debian_docker_luacheck:
 test_debian_install_luacheck:
 	sudo apt update -y
 	sudo apt install -y lua5.1 luarocks
-	sudo luarocks install luacheck 0.25.0
+	sudo luarocks install luacheck
```

Comment:
I thought that linter finding problems is a good thing. We could pin it to current latest version 0.26.1, and increment it when needed/possible, but IMO that's not the best approach, because we will forget to do so, and just stick to 0.26.1. Even if in future it will be able to find problems in our current code we always can add an ignore for that and create corresponding issue to fix the problem with our code later.

Reply:
Fixed, by freezing on 0.26.1.